create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -97,20 +107,34 @@
     private static final int CONTRIBUTOR_151_SEQUENCE_NUMBER = 151;
     private static final String FILENAME_EXPECTED_ABSTRACT_IN_0000469852 = "expectedAbstract.txt";
     private static final String EXPECTED_ABSTRACT_NAME_SPACE = "<abstractTp";
+    private FakeS3Client s3Client;
+    private S3Driver s3Driver;
+    private ScopusHandler scopusHandler;
+    private WireMockServer httpServer;
+    private URI serverUri;
+    private MetadataService metadataService;
 
     @BeforeEach
     public void init() {
         s3Client = new FakeS3Client();
         s3Driver = new S3Driver(s3Client, "ignoredValue");
-        scopusHandler = new ScopusHandler(s3Client);
+        startWiremockServer();
+        HttpClient httpClient = WiremockHttpClient.create();
```

Comment:
Specialized HttpClient that can deal with Https requests, copied from doi-registrar.